lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -60,7 +60,6 @@
             this.$btn_rescore_problem_if_higher_all = this.$section.find("input[name='rescore-problem-all-if-higher']");
             this.$btn_task_history_all = this.$section.find("input[name='task-history-all']");
             this.$table_task_history_all = this.$section.find('.task-history-all-table');
-            this.$btn_download_blank_lti = this.$section.find("input[name='lti-blank-grades']");
```

Comment:
I moved the declaration and event binding down below,
to minimize the diff and group our fork.

Reply:
oh nice